src/pages/docs/chat/rooms/index.mdx

```diff
@@ -159,7 +160,7 @@ By default the `ChatRoomProvider` will automatically call [`release()`](https://
 
 ## Attach to a room <a id="attach"/>
 
-As soon as a client is attached to a room, Ably will begin streaming messages and events to them, regardless of whether or not they have registered any listeners to receive those messages and events.
+To start receiving messages and events from a room, you need to attach to it. Attaching to a room tells Ably to make sure the room is created server-side, starts streaming messages to the client, and ensures that events are not missed in case of temporary network interruptions.
```

Comment:
Do we want to use the word `created` here? Might imply some explicit creation/deletion of rooms (which is quite standard in other providers).

Reply:
Rewording to avoid saying anything about server-side state